src/Diagnostics.RuntimeHost/Program.cs

```diff
@@ -41,32 +42,37 @@ public static IWebHost BuildWebHost(string[] args)
             return WebHost.CreateDefaultBuilder(args)
                 .ConfigureAppConfiguration((context, config) =>
                 {
-                    var (keyVaultConfig, keyVaultClient) = GetKeyVaultSettings(context, config);
+                    var (keyVaultUri, keyVaultClient) = GetKeyVaultSettings(context, config);
 
                     config
                         .AddAzureKeyVault(
-                            $"https://{keyVaultConfig}.vault.azure.net/",
+                            keyVaultUri,
                             keyVaultClient,
                             new DefaultKeyVaultSecretManager())
                         .AddEnvironmentVariables()
                         .AddCommandLine(args)
                         .Build();
                 })
+                .ConfigureLogging(loggingBuilder =>
+                {
+                    //Send log output to text file application logs
+                    loggingBuilder.AddAzureWebAppDiagnostics();
```

Comment:
Correct me if I'm wrong here, but is it a good practice to have application logs in text file in the file system as they can take up space?

Reply:
There is a size limit and when its reached it just starts cleaning away older files. Also it honors the app service settings, so if we disable in the portal then it won't write logs. https://github.com/Azure/Azure-AppServices-Diagnostics/pull/397/files#diff-9d6691fbc408270322f673ad60cb0c82

Reply:
Any reason for using this logging instead of relying on kusto and/or appinsights which are way more scalable?

Reply:
Unfortunately AppInsights is not easily consumable in these clouds at the moment and we need to make changes to make our subscription and app a first party app to get information sent to Kusto. This was the quickest way to get a sense of what is going on in the system while we get all the other pieces together to make it send log data to Kusto.